index.d.ts

```diff
@@ -57,13 +58,33 @@ declare module "oncoprintjs"
         // either `colormap_name` or `colors` needs to be present
         colors?: string[]|number[][]; // hex, rgb, rgba | [r,g,b,a][]
         colormap_name?: string; // name of a colormap found in src/js/heatmapcolors.js
+        value_stop_points: number[];
         null_color?: string;
 
         log_scale?:boolean;
         value_key: string;
         value_range: [number, number];
     }
 
+    // TODO: it would be more elegant to create multiple inheritance (if possible) since
+    // IGradientAndCategoricalRuleSetParams is a IGradientRuleSetParams and 
+    // ICategoricalRuleSetParams with a different `type` field.
+    export interface IGradientAndCategoricalRuleSetParams extends IGeneralRuleSetParams {
+        type: "gradient+categorical";
+        // either `colormap_name` or `colors` needs to be present
+        colors?: string[]|number[][]; // hex, rgb, rgba | [r,g,b,a][]
```

Comment:
theres a failure of documentation from the existing code. it's just wrong. the type of colors should be `[number, number, number, number][]`

Reply:
could you please change that here and in the typedef of GradientRuleSet

Reply:
Thats ok. To be sure hex-color strings are not supported? So the line becomes:

`colors?: [number, number, number, number][]`

?

Reply:
yes it seems to be that way

Reply:
Ok, I keep as proposed.